script/Arcueid/Day 5/05_00_ARC05_1.txt

```diff
@@ -178,22 +178,22 @@ It seems that I fell asleep on the floor.
 [sha:4028002da963458fe0117201bfb32cb0fa39f550]{
 -- Page 15, Offset 8626.
 -- なぜかカーテンが開いていて、体にはシーツがかぶせられている。
-For some reason, the curtains are open and there is a bedsheet covering my body.
+For some reason, the curtains are open and there's a bedsheet covering my body.
 }
 [sha:836e56d307eeecf281fa30a4f46dedf82ebc5b8a]{
 -- Page 16, Offset 8627.
 -- 時刻は昼を過ぎたあたり。
-The time is just past noon.
+It's just past noon.
 }
 [sha:50c00909bb7306eb8b9710841071070c4ddd1ec2]{
 -- Page 16, Offset 8628.
 -- ベッドにアルクェイドの姿はなく、つけっぱなしのテレビがつまらないニュースを流していた。
-There's no sign of Arcueid on the bed, and the TV has been left broadcasting a steady stream of dull news to nobody in particular.
+There's no sign of Arcueid on the bed, and the TV's been left on, broadcasting some dull news.
```

Comment:
Kinda like the ambiance of the original imo

Reply:
the "to nobody in particular" part feels a bit overdone imo, how bout just 

"There's no sign of Arcueid on the bed, and the TV has been left broadcasting a steady stream of dull news."

Reply:
I like it because I feel it accentuates the つけっぱなし a bit, like it has been left on but there's explicitly no audience [Arcueid has left]. I won't die on this hill though.